Automattic/wp-calypso
# A4A > Marketplace: Implement animation for the illustration background when switching tabs

Closes https://github.com/Automattic/jetpack-genesis/issues/456

## Proposed Changes

* This PR implements animation for the illustration background for the new hosting page when switching tabs.

## Why are these changes being made?

* This change is mainly for the new hosting page, includes animation for the background. 

## Testing Instructions

* Open the A4A live link
* Visit Marketplace - Hosting. Verify you can see the animation when switching between tabs:


https://github.com/user-attachments/assets/7659d405-b0fb-4fcc-9fb6-0b1ad8eb93b4



## Pre-merge Checklist
